README.md

```diff
@@ -1,19 +1,118 @@
-# LangfuseSdk
+# Langfuse SDK
 
-Langfuse SDK with generated API using [open-api-generator](https://github.com/aj-foster/open-api-generator).
-You can read the API documentation over here: https://api.reference.langfuse.com/.
+**NOTE:** This is an **unofficial** Langfuse SDK client for Elixir, developed and maintained by [Workera](https://workera.ai/).
 
-## Settings
+[Langfuse](https://github.com/langfuse/langfuse) is an open-source platform that helps teams efficiently debug, analyze, and optimize large language model (LLM) applications by providing detailed execution insights.
+You can read the API documentation over here: <https://api.reference.langfuse.com/>.
 
-Go to Langfuse API Keys inside organization settings and then export the following variables to your shell profile:
+## Installation
 
+Add Langfuse to your `mix.exs` dependencies:
+
+```elixir
+def deps do
+  [
+    {:langfuse_sdk, "> 0.0"}
+  ]
+end
+```
+
+Fetch the dependency with:
+
+```shell
+mix deps.get
 ```
-export LANGFUSE_SECRET_KEY="your secret key..."
-export LANGFUSE_PUBLIC_KEY="your public key..."
+
+## Configuration
+
+In your `config/config.exs`, set the following:
+
+```elixir
+config :langfuse_sdk,
+  host: System.get_env("LANGFUSE_HOST"),
+  secret_key: System.get_env("LANGFUSE_SECRET_KEY"),
+  public_key: System.get_env("LANGFUSE_PUBLIC_KEY"),
+```
+
+You can find the secret and public key in the `organizations` setting in Langfuse.
+
+## Usage
```

Comment:
@nickgnd Would it be okay with you if we remove the usage section? I think the Langfuse API does a good job explaining how to interface with the objects. So, I would rather focus on documenting the specifics of our API (which are not that different tbh).

Reply:
I'd personally keep it, I think having a "complete" example would be useful and make its integration easier.